db/dbOps.js

```diff
@@ -58,22 +61,36 @@ async function createTables() {
 			ip INET
 		);
 	`);
+	await pool.query(`
```

Comment:
Given that the pairing of country codes and country names is "static" once we have downloaded the JSON and will not change, it is not entirely clear to me what's the benefit of storing it on the database. You can import it anywhere in the code where you need the pairing by importing the json file. The database only needs to store the `country_code` in table `locations`. 

Can you make the case for why should we store it in the DB?

Reply:
Good point. I think we should absolutely place it in a JSON file and import it. I'll look into doing that in this architecture.